assets/src/edit-story/components/canvas/layout.js

```diff
@@ -102,6 +102,21 @@ const PageAreaFullbleedContainer = styled(Area).attrs({
   display: flex;
   justify-content: center;
   align-items: center;
+
+  ${({ isBackgroundSelected, theme }) =>
+    isBackgroundSelected &&
+    `
+    &:before {
+      content: '';
+      position: absolute;
+      top: -4px;
+      left: -4px;
+      right: -4px;
+      bottom: -4px;
+      border: ${theme.designSystemTheme.colors.blue[20]} 1px solid;
```

Comment:
Should be updated to `border.focus` - @samitron7 is correcting it.

Reply:
We ended up deciding that we'll add a now border property named `border.selection` - it'll be the same color as focus, but more properly named (and could be changed independently), so please update to that once @BrittanyIRL merges.

Reply:
PR up #6271